media/src/utils.js

```diff
@@ -1187,6 +1187,30 @@ class ParametricCurve extends THREE.Curve {
     }
 }
 
+class ParametricSurface extends THREE.Object3D {
```

Comment:
Yeah, a 3Demos object doesn't map perfectly onto a Three.js Object3D, so this doesn't really make sense: each 3Demos object is made of a collection of Three.js Object3Ds. Also, if you want to propose a refactor like this, it should happen separately from stuff like poll functionality.

Reply:
Right. Forgot to remove that. This was a sort of attempt to handle the `<svelte:component>` but it was causing trouble so I scrapped it.